### PR TITLE
Add "format" target

### PR DESCRIPTION
**Pull request by chanseokoh**
Sometimes I want to do a quick formatting.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -32,13 +32,13 @@ while getopts qe c; do
   \?) usage;;
   esac
 done
-shift `expr $OPTIND - 1`
+shift $( expr $OPTIND - 1 )
```
Comment (TadCordle) [inline build.sh]: Newby question: what's the difference?

File: build.sh [hunk 2]
```diff
@@ -32,13 +32,13 @@ while getopts qe c; do
...
 
 if [ $# -eq 0 ]; then
   set -- core plugins gradle maven
 fi
 
-set -e  # exit on error
+set -o errexit  # exit on error
```
Comment (TadCordle) [inline build.sh]: Second newby question: what's the difference here as well?

Comment (chanseokoh) [reply build.sh]: Same, `-e` is just a shortcut.

File: build.sh (hunk 1 above)
Comment (chanseokoh) [reply build.sh]: They are same, but `$( )` has the advance that you can nest. According to http://mywiki.wooledge.org/BashFAQ/082, `$( )` is slightly preferred.

**Pull request merged**